test/RToken.test.js

```diff
@@ -1775,12 +1775,12 @@ contract("RToken", accounts => {
         assert.equal(wad4human(await rToken.receivedSavingsOf.call(customer1)), "100.00000");
 
         await web3tx(rToken.changeAllocationStrategy,
-            "change allocation strategy", {
+            "change allocation strategy 1st time", {
                 inLogs: [{
                     name: "AllocationStrategyChanged",
                     args: {
                         strategy: compoundAS2.address,
-                        conversionRate: toWad(0.1)
+                        conversionRate: "99999999999999995"
```

Comment:
whats up here?

Reply:
The exchange rate is affected by the saving assets value, it changes if we wait for more blocks. I don't know why it was changed from 0.1wad tbh, something related to the upgrade of ganache my guess is.

Although I have found some issue of current changeAllocationStrategy implementation, will make it a target of 1.1 to fix this before we actually using it.

Thanks for the review